### ISSUE TITLE
Requirements/Setup Issues

### ISSUE THREAD

**Issue by Daynar100** [lokkelvin2/dc_tts#7]
Just thought I'd let you know in order to get this working I had to downgrade from Python 3.8 to 3.7 so I could use a tensorflow version < 2.0 as it uses a bunch of removed features. I also had to downgrade numba to 0.43.0. It also is unable to find cudart64_100.dll but that's optional so it isn't a huge deal. 

PS How do I format my transcript file for training. I was looking at some of the examples and it looked liike they formatted it differently and I couldn't find any clear documentation on how it should be formatted.

PPS I messaged you on reddit first, but I figured I'd make an issue as well.

**Comment by lokkelvin2** [comment]
Thanks for pointing that out.

I believe the cudart64_100.dll error is related to the installation of cuda. For tensorflow_gpu-1.13.1 (what I used), you need to install CUDA 10.0 and cuDNN 7.4. Note that different versions have compatibility issues with one another so you need to stick with the [official recommended versions](https://www.tensorflow.org/install/source#tested_build_configurations). If you are installing on windows, CUDA 10.0 also requires visual studio 2017 to be installed (see [here](https://docs.nvidia.com/cuda/archive/10.0/cuda-installation-guide-microsoft-windows/index.html)).

In the dc_tts repo the author provided two methods of processing training data. I used the method for the LJ speech dataset, which means that any custom transcript.csv will need to follow the metadata convention in the LJ dataset. The format is filename|transcript|normalized_transcript, where normalized transcripts have numerals and symbols fully in words.

For example, this is a line from the LJ dataset:
> LJ002-0009|On the 27th April, in the following year,|On the twenty-seventh April, in the following year,

**Comment by Daynar100** [comment]
Okay, cool. Thank you my dude. I got a little confused as to what I actually needed to do since the LJ dataset had different formatting than the Bible one. Those were the 2 I looked at.
